### PR TITLE
wgpu: Prefix WGPU labels with "vello."

### PR DESCRIPTION
**Pull request by waywardmonkeys**
This makes them easier to identify as being from Vello when embedding into an application.

### CODE REVIEW

**Comment by waywardmonkeys** [comment]
For example, in Apple's tools:

image

But when you have your own application and a bunch of pipelines, then it is harder to identify who something belongs to without it being clear in the name.

**Review by DJMcNab — APPROVED** [review]
Would it be worth automatically stripping these prefices in `with_winit` for display?

Otherwise, seems reasonable. Should we also apply this for buffers as well?

**Comment by waywardmonkeys** [comment]
Maybe and maybe. I confess to feeling lazy about the first. The second, less so ... but I hadn't run into it yet in the tools. :) Will think about it for a couple of hours!

**Comment by DJMcNab** [comment]
Laziness about the first is very reasonable, fwiw.

**Comment by waywardmonkeys** [comment]
I changed this to use `vello.` as a prefix instead of `vello_`.

I updated it to handle various buffers as well.

For debug stuff, I prefixed with `vello.debug.`

In some cases, I changed a space character to an underscore.

File: vello/src/render.rs [hunk 1]
```diff
@@ -178,9 +178,9 @@ impl Render {
...
             packed.resize(size_of::<u32>(), u8::MAX);
         }
-        let scene_buf = ResourceProxy::Buffer(recording.upload("scene", packed));
+        let scene_buf = ResourceProxy::Buffer(recording.upload("vello.scene", packed));
         let config_buf = ResourceProxy::Buffer(
-            recording.upload_uniform("config", bytemuck::bytes_of(&cpu_config.gpu)),
+            recording.upload_uniform("vello.config", bytemuck::bytes_of(&cpu_config.gpu)),
         );
         let info_bin_data_buf = ResourceProxy::new_buf(
             buffer_sizes.bin_data.size_in_bytes() as u64,
```
Comment (DJMcNab) [inline vello/src/render.rs:186]: There are lots of other calls to `new_buf`; why have you changed only the ones you have?

Comment (waywardmonkeys) [reply vello/src/render.rs:186]: because I wasn't looking at new_buf but `upload` / `upload_uniforms` ... I'll hit the others now, I guess. Thanks!

Comment (waywardmonkeys) [reply vello/src/render.rs:186]: Fixed and force pushed.

**Comment by waywardmonkeys** [comment]
One more thing it seems ... looking at it now under the Metal tools.

**Review by DJMcNab — APPROVED** [review]
This will accrue merge conflicts, and seems fine to land

**Pull request merged**